x-pack/plugins/fleet/public/applications/fleet/sections/agent_policy/create_package_policy_page/index.tsx

```diff
@@ -60,6 +60,11 @@ const StepsWithLessPadding = styled(EuiSteps)`
   .euiStep__content {
     padding-bottom: ${(props) => props.theme.eui.paddingSizes.m};
   }
+
+  // compensating for EuiBottomBar hiding the content
+  @media (max-width: 767px) {
```

Comment:
I think we can use breakpoints from the theme `euiBreakpoints`

Reply:
fixed